templates/systemd-override.erb

```diff
@@ -1,4 +1,8 @@
+<% if @managed_package_repo -%>
+  .include /lib/systemd/system/postgresql-<%= @version %>.service
```

Comment:
@igalic this template is only used by [EL7](https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/manifests/server/config.pp#L160-L177) and EL7 is only supported in >= 9.3
I've confirmed 9.3 and 9.4 available packages have this renamed library:
```
# rpm -qlp postgresql93-server-9.3.5-2PGDG.rhel7.x86_64.rpm | grep systemd
/usr/lib/systemd/system/postgresql-9.3.service
# rpm -qlp postgresql93-server-9.3.6-2PGDG.rhel7.x86_64.rpm | grep systemd
/usr/lib/systemd/system/postgresql-9.3.service
# rpm -qlp postgresql94-server-9.4.0-1PGDG.rhel7.x86_64.rpm |grep systemd
/usr/lib/systemd/system/postgresql-9.4.service
# rpm -qlp postgresql94-server-9.4.1-1PGDG.rhel7.x86_64.rpm |grep systemd
/usr/lib/systemd/system/postgresql-9.4.service
```